hpx/runtime/threads/policies/local_priority_queue_scheduler.hpp

```diff
@@ -247,7 +247,7 @@ namespace hpx { namespace threads { namespace policies
                 num_pending_misses += high_priority_queues_[num_thread].data_->
                     get_num_pending_misses(reset);
             }
-            if (num_thread == 0)
+            if (num_thread == num_queues_-1)
```

Comment:
Does this make a difference?

Reply:
There was some inconsistency that we were accessing the low-priority queue from different cores, sometimes it was core zero sometimes the last one. I made it consistent to reduce false  sharing on that queue.